### DISCUSSION TITLE
Code blocks in reply sections are a lot brighter than default

### DISCUSSION THREAD

**Discussion by Andre601** [community/community#14419] [Discussions]
Recently noticed that code blocks in comments to replies are a lot brighter compared to when used in replies themself. I observed this while using the default dark theme.

This color change isn't really needed in my opinion as the default code block color (`--color-canvas-subtle`) is still visible enough and the usage of `--color-neutral-muted` is really not needed.

The comment below this discussion shows this difference, but for clarity (And in case this gets changed in the future) is here a screenshot showing this:

![image](https://user-images.githubusercontent.com/11576465/162036876-067e7515-3ffd-4d6e-8f7d-91fbf3cf2c8e.png)

And for comparison is here the same comment, but with the bottom code block having its color changed to `--color-canvas-subtle`:

![image](https://user-images.githubusercontent.com/11576465/163068712-dee9e053-41ae-4699-a3e5-b717e022e523.png)

**Comment by Andre601** [comment]
Example Code block reply:

```markdown
# Title

Text

![image](/cat.png)
```

**Reply by Andre601** [reply]
Example Code block reply comment:

```markdown
# Title

Text

![image](/cat.png)
```

**Comment by Andre601** [comment]
Quickly updated my comment to be a bit more clear and to also show how the code box would look like if it used the exact same color.

I personally still believe that this is a bug, because I can't see any reason - accessibility or otherwise - that would explain the usage of this somewhat more saturated color. The background of the comment isn't getting brighter after all, but darker, so the default canvas color would be even more visible anyways. It also makes the text a bit harder to read.

**Comment by francisco-camargo** [comment]
Just noticed this and I find it very distracting. Hope there is a workaround!
